compiler/test/compilable/extra-files/json.json

```diff
@@ -184,7 +184,7 @@
                 "protection": "public"
             },
             {
-                "char": 11,
+                "char": 1,
```

Comment:
What does this field mean? How do we now that it is correct to change this test? Please accompany such test changes with explanations.

Reply:
Ok, so this is the offset from the start of the line.